dockerize-repository/docker-compose.yml

```diff
@@ -12,9 +12,7 @@ services:
             - "5432"
 
     b2share:
-        build:
-            context: ..
-            dockerfile: dockerize/Dockerfile
+        image: eudatb2share/b2share:2.0.1
```

Comment:
I see that EUDAT-B2SHARE/dockerize is empty. Maybe this docker-compose file is intended to be there?

If yes, I would still keep one docker-compose file either in this repo or the other just for testing local code in docker.